### PR TITLE
deploy with ansible

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This PR clean up docker compose files and push devspace container management to ansible playbook, see relevant PR https://github.com/openmicroscopy/infrastructure/pull/114 for deployment

Note: devspace can only be deployed on Linux. There are significant performance issues when writing to the mounted volume on Mac, see https://forums.docker.com/t/file-access-in-mounted-volumes-extremely-slow-cpu-bound/8076/

### CODE REVIEW

**Comment by jburel** [comment]
* Changes made to the description of PR https://github.com/openmicroscopy/infrastructure/pull/114
should be propagated to the README
 * a typo i missed ``Remot`` instead of ``Remote``

**Comment by aleksandra-tarkowska** [comment]
Readme updated as in https://github.com/openmicroscopy/infrastructure/pull/114

File: .travis.yml [hunk 1]
```diff
@@ -7,26 +7,21 @@ services:
...
-  - sudo DEBIAN_FRONTEND=noninteractive apt-get -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confnew" install docker-engine=${DOCKER_VERSION}
-  - export USER_ID=`$UID`
-  - docker -v
-
   - sudo rm /usr/local/bin/docker-compose
   - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
   - chmod +x docker-compose
   - sudo mv docker-compose /usr/local/bin
   - docker-compose -v
+  - export USER_ID=`$UID`
```
Comment (manics) [inline .travis.yml]: What do the backticks (`` ` ``) do here?

**Comment by joshmoore** [comment]
Merging in sysadmin meeting followed by openmicroscopy/infrastructure#114. Tested by @jburel.

File: .travis.yml (hunk 1 above)
Comment (joshmoore) [inline .travis.yml]: I see what you mean. Assuming this will be corrected in a follow-up.

**Pull request merged**